src/views/components/checkbox/index.njk

```diff
@@ -0,0 +1,29 @@
+---
+title: Checkbox
+description: Checkboxes let the user select one or more options from a list.
+layout: default
+eleventyNavigation:
+  key: Checkbox
+  section: components
+---
+{% from "example.njk" import example %}
+
+<section>
+    <p>Checkboxes let the user select one or more options from a list.</p>
+    {{ example({group: "components", item: "checkbox", example: "default", size: "m", html: true, open: false }) }}
+</section>
+
+<section class="my-8">
+    <h2 class="mb-4">When to use this component</h2>
+    <p>Use the input component when you need to let users enter alphanumeric information that is no longer than one line, such as their name, telephone number or email address.</p>
```

Comment:
Seems to be an `input` description not a `checkbox` one.

Reply:
Good spot 🤦🏽  have updated